com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -22,7 +22,8 @@ Additional documentation and release notes are available at [Multiplayer Documen
 
 - Fixed issues when multiple `ConnectionApprovalCallback`s were registered (#1972)
 - Fixed endless dialog boxes when adding a NetworkBehaviour to a NetworkManager or vice-versa (#1947)
-- `FixedString` types can now be used in NetworkVariables and RPCs again without requiring a `ForceNetworkSerializeByMemcpy<>` wrapper (#1961)
+- `FixedString`, `Vector2Int`, and `Vector3Int` types can now be used in NetworkVariables and RPCs again without requiring a `ForceNetworkSerializeByMemcpy<>` wrapper (#1961)
```

Comment:
Mentioning Vector2Int and Vector3Int was supposed to be part of my last PR but I forgot to push it after I committed it so it ended up in this branch by accident, but also needs to be there... I'll move it into a separate PR if desired but it seems so small as to not warrant it, maybe?